[get, subset] chore: skip existing main (#194)
mercator-ocean/copernicus-marine-toolbox: doc/usage/shared-options.rst

.. note::
    For versions <=1.2.4, all logs are sent to stdout by default.

Option ``--skip-existing``
*******************************

> This option allows you to skip the download of files that already exist in the output directory. If the file exists, the download will be skipped, and the file will not be overwritten.

Reviewer: ```suggestion
This option allows you to skip the download of files that already exist at the output destination. If the file exists, the download will be skipped.
```

Remember that by default, the toolbox creates a new file with an index (eg 'filename_(1).nc').

> See in :ref:`Response types documentation <response-types>` the ``status`` and ``message`` fields to understand the status of your request.

Reviewer: ```suggestion
See in :ref:`Response types documentation <response-types>` the ``status`` and ``message`` fields for more information about request statuses.
```

> .. note::
>     This option is incompatible in ``get`` command with the ``--sync``, ``--sync-delete``, ``--no-directories`` and ``--overwrite-outputdata`` option.
> 
>     For the ``subset`` command, it is incompatible with the ``--overwrite-outputdata`` option.

Reviewer: ```suggestion
.. note::
    This option is incompatible with the ``--overwrite-output-data`` option and the ``--sync`` and ``--sync-delete`` options.
```
